src/multipart/related.rs

```diff
@@ -1,38 +0,0 @@
-//! Helper functions for multipart/related support
```

Comment:
I have moved the functions into the mod.rs file of the new directory (as mentioned in the commit message). I thought this would have the same effect as having the file as it was before.